packages/experimental-utils/src/index.ts

```diff
@@ -1,8 +1,3 @@
-import * as ASTUtils from './ast-utils';
-import * as ESLintUtils from './eslint-utils';
-import * as JSONSchema from './json-schema';
-import * as TSESLint from './ts-eslint';
-import * as TSESLintScope from './ts-eslint-scope';
+// Once typescript-eslint hits v6, this package will console.warn to switch...
```

Comment:
Nit: I think it would improve discovery to format this as a traditional `TODO` comment